.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.8"]
-        toxenv: [django32, quality, pii_check]
+        toxenv: [django32, quality, pii_check, check_keywords]
```

Comment:
It would be helpful if we could see what words were reserved in the check output.